python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -1142,6 +1157,27 @@ def submit_run_request(
                     )
                     updated_evaluation_keys.add(entity_key)
 
+            now = get_current_timestamp()
+            if now - last_status_poll_time >= poll_interval:
+                if remote_sensor:
```

Comment:
this check means it won't do anything in the non-sensor AMP case, which may be fine considering we are actively moving away from it. If we wanted to include it, it would need to instead to do this check to see if the global AMP toggle is paused: https://github.com/dagster-io/dagster/blob/0ff03a6f3865f7ed61db57c093ac3cc361862282/python_modules/dagster/dagster/_daemon/asset_daemon.py#L445